src/components/form/PhoneInquiryForm.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from 'react';
+import { FormProvider, useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { phoneInquiryValidationSchema } from '../../utils/validationSchema';
+import PersonalInformationSection from './sections/PersonalInformationSection';
+import ResidenceStatusSection from './sections/ResidenceStatusSection';
+import LanguageSection from './sections/LanguageSection';
+import SupportNeedsSection from './sections/SupportNeedsSection';
+import AsylumProcedureSection from './sections/AsylumProcedureSection';
+import MaritalStatusSection from './sections/MaritalStatusSection';
+import ChildrenInformationSection from './sections/ChildrenInformationSection';
+import EmploymentSection from './sections/EmploymentSection';
+import FormNavigation from './FormNavigation';
+
+interface PhoneInquiryFormProps {
+  onSubmit: (data: any) => void;
```

Comment:
Probably the type should be `PhoneInquiryFormData`, you have this also on the App.tsx component